src/components/widgets/BaseInput.js

```diff
@@ -22,14 +22,18 @@ function BaseInput(props) {
   const _onChange = ({ target: { value } }) => {
     return props.onChange(value === "" ? options.emptyValue : value);
   };
+
+  const cleanProps = { ...inputProps };
+  delete cleanProps.rawErrors;
```

Comment:
I think you can do `const {rawErrors, ...cleanProps} = inputProps;` See the "Rest in object destructuring section" of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring.